stack/global/s3-private.tf

```diff
@@ -1,4 +1,4 @@
-module "s3-storage" {
```

Comment:
NIT: Got it now, thanks for your comment; if we also use it for Terraform state, maybe just call it "global-buckets" and leave a comment with examples why we want these?